### ISSUE TITLE
How do we elevate the fair theme to take advantage of VR platform?

### ISSUE THREAD

**Issue by JacobChrist** [VRCreatorAcademy/vr-creator-academy-collab#4]
I had an epiphany tonight.  Something had been bothering me about the fair theme since it was announced, and tonight I realized what it is.  VR gives us the ability to be transported to worlds not possible in real life and a fair is, well, probably everyone has been to a fair at one time or another.  With that being said, I also like constraints so how do we elevate the fair theme to take advantage of VR platform? Replicating a fair in VR is I think is probably a great vehicle as a learning tool, but not inspirational.  Having stuck around on some calls it seems like many of the people hanging around here are motivated by inspirational.

**Comment by SilverWyzardess** [comment]
LOVE the idea of a Steampunk fair--or perhaps a "many worlds" fair?  Where a contraption/portal transports people to Victorian times or the 1920s or Europe during the Renaissance or?  And what if certain aspects of the Fair gave you new capabilities?   Maybe you can fly in certain areas.  Maybe an aardvark comes up to you and offers you a ride.   Maybe part of the fair is even "Dreamworld" where you're on a train platform one minute and skydiving the next?  Just random thoughts.

**Comment by JacobChrist** [comment]
@SilverWyzardess great ideas... I'm really liking the start small and build out.  I'm hoping once we get the first level going that this will help elevate the project and creative talent working on it.

**Comment by SilverWyzardess** [comment]
Of course, now I have to finish my GitHub setup with my "techspert" this
week so I can start showing you what I'm making!
